### PR TITLE
Benchmark support

### LINKED ISSUE #72: Add benchmark feature preparation
- [x] Fix the compilation error with runtime-benchmarks feature
- [x] Add sub command benchmark in parachain
- [x] Enable benchmark feature in runtime
- [x] Enable and run benchmark for account linker
- [x] Enable and run benchmark for offchain worker

### PR DESCRIPTION
**Pull request by suinuj**
- enable the compilation with runtime-benchmarks feature
- add sub command to run benchmark
- add benchmark for both account linker and offchain worker in runtime

### CODE REVIEW

**Comment by h4n0** [comment]
Please don't forget linking this with the corresponding issue next time

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -934,6 +934,42 @@ impl_runtime_apis! {
...
+	impl frame_benchmarking::Benchmark<Block> for Runtime {
+		fn dispatch_benchmark(
+			config: frame_benchmarking::BenchmarkConfig
+		) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
+			use frame_benchmarking::{Benchmarking, BenchmarkBatch, add_benchmark, TrackedStorageKey};
+
+			use frame_system_benchmarking::Pallet as SystemBench;
+			impl frame_system_benchmarking::Config for Runtime {}
+
+			let whitelist: Vec<TrackedStorageKey> = vec![
```
Comment (Satoshi-Kusumoto) [inline runtime/src/lib.rs:948]: What is this

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -126,3 +132,16 @@ std = [
 	"log/std",
 	"max-encoded-len/std",
 ]
+
+runtime-benchmarks = [
```
Comment (Satoshi-Kusumoto) [inline runtime/Cargo.toml:136]: What are the selection criterions here?

File: runtime/src/lib.rs (hunk 1 above)
Comment (suinuj) [reply runtime/src/lib.rs:948]: code from upstream, these variables needed to build a batches as returned value.

File: runtime/Cargo.toml (hunk 2 above)
Comment (suinuj) [reply runtime/Cargo.toml:136]: just put the system and three litentry pallets into list.

**Review by Satoshi-Kusumoto — APPROVED** [review]
Then you should have an issue about how to select pallets
If I ask you something you don't know, either convince me it's not important, or write an issue

**Comment by suinuj** [comment]
> Then you should have an issue about how to select pallets
> If I ask you something you don't know, either convince me it's not importance, or write an issue

will write an issue to clarify the requirement for benchmarks testing before network launch.

**Pull request merged**